load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -175,41 +179,39 @@ func (s *relaysSuite) mapSessionInfoForLoadTestDurationFn(
 
 			// The test is not to be started yet, skip the notification to the downstream
 			// observables until the first block of the next session is reached.
-			return nil, true
+			return nil, skipNotifyDownstreamSteps
 		}
 
 		// If the test has not started, set the start block height to the current block height.
 		// As soon as the test starts, s.startBlockHeight will no longer be updated.
 		// It is updated only once at the start of the test.
 		if waitingForFirstSession {
 			// Record the block height at the start of the first session under load.
-			s.testStartBlockHeight = blockHeight
+			s.testStartHeight = blockHeight
 			// Mark the test as started.
 			waitingForFirstSession = false
 
-			logger.Info().Msgf("Test starting at block height: %d", s.testStartBlockHeight)
+			logger.Info().Msgf("Test starting at block height: %d", s.testStartHeight)
 		}
 
 		// If the test duration is reached, stop sending requests
-		if blockHeight >= s.testStartBlockHeight+s.testDurationBlocks {
+		if blockHeight >= s.testStartHeight+s.relayLoadDurationBlocks {
```

Comment:
Please move the RHS into a local var with an appropriate name (like on line 202)